h2o-core/src/main/java/water/persist/PersistManager.java

```diff
@@ -38,6 +39,12 @@
    * layer forwards the request through HDFS API. */
   private static final String PROP_ENABLE_HDFS_FALLBACK = SYSTEM_PROP_PREFIX + "persist.enable.hdfs.fallback";
 
+  private static final String PROP_IMPORT_FILES_PATTERNS = SYSTEM_PROP_PREFIX + "persist.importFile.patterns";
+  private static final String PROP_IMPORT_FILES_PATTERNS_DEFAULT = 
```

Comment:
Inconsistent naming. This is no property and is not configurable.

Reply:
yeah, I though "PROP_.._DEFAULT" but I see your point, do you have a suggestion?

wdyt about DEFAULT_IMPORT_FILES_PATTERNS?

Reply:
`IMPORT_FILES_PATTERNS_DEFAULT` or your option, whatever. This is minor thing.

Reply:
thank you, changed the name